app/templates/partials/add-service/step-create-service.html

```diff
@@ -27,9 +29,10 @@ <h2 class="heading-medium">{{ _("Email address") }}</h2>
       {{ textbox(form.email_from, hint=hint_txt, suffix=suffix_txt, width='1-2', maxlength=64) }}
 
       <span id="preview" style="display: none">
```

Comment:
This span should be div to pass HTML validation.

Reply:
Nice!  I think we should also add a tabstop on this information box so that a screen reader user doesn't mistakenly jump over it when tabbing through form fields.

```suggestion
      <div id="preview" style="display: none" tabindex="0">
```

Reply:
done!